Fixes #545

SirTyson opened the pull request:
Resolves issue #545. Now displays a reload button after loading bar has stalled for a certain amount of time. Also removes red version warning box.

dbrgn commented:
Thanks! Some feedback:

- The layout/spacing doesnt't look too nice:

![img](https://tmp.dbrgn.ch/screenshots/20180808094306-w4eh1aw3.png)

Maybe change the min-height in `#main-wrapper:not(.wide) #main` (src/sass/layout/_main.scss) to 620px and add 16px top margin to he button?

![img2](https://tmp.dbrgn.ch/screenshots/20180808094434-n6848fgg.png)

- Maybe we could change the "If you want, you can forget this session." text to be a button right next to the reload button? It could be labelled with "Forget Session".

SirTyson commented:
@dbrgn Thanks for the advice! I'll work on it.

SirTyson commented:
I added the forget session button and cleaned up the UI some:


![new ui](https://user-images.githubusercontent.com/25424712/43859842-7008ccb4-9b17-11e8-8459-f4195854a3af.png)



I also added "FORGET_SESSION_BTN" to de.json, but it will probably have to be changed since I don't know German.

rugk commented on public/i18n/de.json:
```diff
@@ -19,6 +19,7 @@
         "UNLOCK_FAILED_FORGOTTEN": "Haben Sie Ihr Passwort vergessen? W\u00e4hlen Sie \"gespeicherte Sitzung l\u00f6schen\" und setzen Sie ein neues Passwort.",
         "IF_YOU_WANT": "Wenn Sie möchten, können Sie die",
         "FORGET_SESSION": "gespeicherte Sitzung löschen",
+        "FORGET_SESSION_BTN": "gespeicherte Sitzung löschen",
```
"Sitzung Vergessen" without a capital "V", please. I mean it hardly matters, as the text is "uppercased"  here, but yeah…

dbrgn replied:
That would have been title casing. I always forget that it's not applied in German.

SirTyson commented on src/sass/sections/_welcome.scss:
```diff
@@ -110,7 +110,7 @@
     .loading {
         margin-top: 48px;
-
+		
```
Do you want to change the other instance of "forget this session" to a button as well to keep a consistent style, or is it better as a hyperlink? Here's a picture to what I am referring to:

![other use of string](https://user-images.githubusercontent.com/25424712/44116489-497cb0b0-9fd6-11e8-8cc8-156acfddfa2e.png)

dbrgn replied:
Oh, I missed that one, good catch! In that case, leave the old strings as-is and don't convert it to a button.
